app/helpers/ui_helper.rb

```diff
@@ -33,22 +33,31 @@ def label_with_icon(icon, label_tag, position)
     ordered.compact.join.html_safe # rubocop:disable Rails/OutputSafety
   end
 
-  def input_field(form: nil, field_name: nil, label: nil, placeholder: 'Enter text', width: 'w-56', left_icon: nil,
-                  right_icon: nil, type: 'text_field', options: [], value: 'nil')
+  def input_field(form: nil, field_name: nil, text_field_name: nil, radio_field_name: nil, label: nil,
+                  placeholder: 'Enter text', width: 'w-56', height: nil, left_icon: nil, right_icon: nil, type: 'text',
+                  options: [], value: nil, rows: '5', option: nil, html_options: {})
```

Comment:
there are options, option these namings makes the code really bad, it is not very easy to understand the thought process here. html_options is a hash here, other's are array and nil, this is very inconsistent.